server/cmd/api/api/playwright_test.go

```diff
@@ -0,0 +1,58 @@
+package api
+
+import (
+	"context"
+	"testing"
+
+	oapi "github.com/onkernel/kernel-images/server/lib/oapi"
+	"github.com/stretchr/testify/require"
+)
+
+func TestExecutePlaywrightRequest_Validation(t *testing.T) {
+	t.Parallel()
+	ctx := context.Background()
+	svc := &ApiService{}
+
+	tests := []struct {
+		name        string
+		code        string
+		expectError bool
+	}{
+		{
+			name:        "empty code",
+			code:        "",
+			expectError: true,
+		},
+		{
+			name:        "valid code",
+			code:        "return 'hello world';",
+			expectError: false,
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			body := &oapi.ExecutePlaywrightCodeRequest{
```

Comment:
Error: vet: cmd/api/api/playwright_test.go:35:18: undefined: oapi.ExecutePlaywrightCodeRequest

@tembo fix this and run `go vet`  until there are no more errors